packages/wrangler/e2e/deployments.test.ts

```diff
@@ -285,7 +285,6 @@ Uploaded 2 of 3 assets
 Uploaded 3 of 3 assets
 ✨ Success! Uploaded 3 files (TIMINGS)
 Total Upload: xx KiB / gzip: xx KiB
-No bindings found.
```

Comment:
This is an unrelated-to-mixed-mode change that @threepointone suggested, that I thought I'd do while in the same area of the codebase. It removes the `No bindings found.` message unless a dry-run is happening